src/command/cmd_ac.c

```diff
@@ -1330,6 +1330,7 @@ cmd_ac_complete_filepath(const char *const input, char *const startstr, gboolean
             return NULL;
         }
     }
+    // XXX: Why free and then use?
```

Comment:
inpcp is just a char pointer. So we free the last strdup()-ed space and now use inpcp to point to a new location.